Libraries/PullToRefresh/PullToRefreshViewAndroid.android.js

```diff
@@ -67,7 +67,8 @@ var PullToRefreshViewAndroid = React.createClass({
   },
 
   setNativeProps: function(props) {
-    return this.refs[NATIVE_REF].setNativeProps(props);
+    let innerViewNode = this.getInnerViewNode();
+    return innerViewNode && innerViewNode.setNativeProps(props);
```

Comment:
When can `innerViewNode` be null? We should understand what's going on here and make sure that it's valid for setNativeProps to be called when `innerViewNode` is null, or instead fix whatever's calling setNativeProps at the wrong time.